src/main/java/core/mate/academy/service/MachineServiceImpl.java

```diff
@@ -14,11 +14,14 @@
 public class MachineServiceImpl implements MachineService<Machine> {
     @Override
     public List<Machine> getAll(Class<? extends Machine> type) {
-        List<? extends Machine> machines =
-                type == Bulldozer.class ? new BulldozerProducer().get()
-                : type == Excavator.class ? new ExcavatorProducer().get()
-                : type == Truck.class ? new TruckProducer().get()
-                : Collections.emptyList();
+        if (type != Bulldozer.class && type != Excavator.class && type != Truck.class) {
+            return Collections.emptyList();
+        }
```

Comment:
Actually, this is not a scalable code. Now, when there are more Cars in the app, you will need to change this expression also, and it will become longer and longer.